dist/audit.h

```diff
@@ -30,14 +30,62 @@
  * NUL-terminated string, while the value is explicitly sized with
  * VALUE_SIZE
  */
-# define CRYPTO_AUDITING_BLOB_DATA(key_ptr, context, value_ptr, value_size)	\
+# define CRYPTO_AUDITING_BLOB_DATA(context, key_ptr, value_ptr, value_size)	\
```

Comment:
Are there compatibility concerns?

Reply:
I think nobody has actually started using this macro, and the file is provided as an example, so it should be safe I would say.